app/helpers/image-location-map.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function imageLocationMap(params/*, hash*/) {
+  // This is a hack that should not make it to production. Its in place until the api for catalog is integrated
+  var newLoc = params.split('v1');
+
+  return newLoc.join('catalog');
+}
+
+export default Ember.Handlebars.makeBoundHelper(imageLocationMap);
```

Comment:
I know this file probably won't survive into release but this form is deprecated and will complain once you rebase..

```export default Ember.Helper.helper(function(params, options) {
...
})```